tests/e2e/common/scripts/trigger-maintenance.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -e
+
+type="$1"
+name="$2"
+to_be_found="$3"
+
+ns=$(kubectl -n "$NAMESPACE" get "$type" "$name" -oyaml | yq -r '.status.instanceNamespace')
+
+kubectl -n "$ns" delete job --ignore-not-found=true --now=true vacuum databasesrepack
```

Comment:
Did you test against the lab? I'm pretty sure you'll run into issues as the e2e user doesn't have the permissions for jobs. You need to add them in `rbac_e2e.jsonnet`.